Xamarin.Forms.Core/Cells/Cell.cs

```diff
@@ -19,6 +19,9 @@ public abstract class Cell : Element, ICellController, IFlowDirectionController
 
 		bool _nextCallToForceUpdateSizeQueued;
 
+		object _forceUpdateSizeLocker = new object();
+		Task _forceUpdateSizeDelayTask = Task.CompletedTask;
```

Comment:
Compile error here:

Cells\Cell.cs(23,41): error CS0117: 'Task' does not contain a definition for 'CompletedTask' [C:\agent\_work\2\s\Xamarin.Forms.Core\Xamarin.Forms.Core.csproj]

Reply:
Fixed this error by replacing Task.CompletedTask by Task.FromResult(true)